public/assets/javascript/app.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function(){
```

Comment:
prefer arrows for simple functions, especially if 'this' is not involved.
$(document).ready(() => $('.parallax').parallax())